management-core/src/test/java/com/ns/task/services/ProductCoreServiceImpTest.java

```diff
@@ -81,14 +80,17 @@ public void insertProduct() throws DataIntegrityViolationException {
         assertProduct(returned, productEntity);
     }
 
-    @Test(expected = ProductManagementException.class)
+    @Test
     public void shouldThrowAnExceptionWhenInsertDuplicateProduct() {
         ProductDTO productDTO = new ProductDTO();
         productDTO.setName("SCREWDRIVER");
         productDTO.setModel("S090");
         productDTO.setPrice(new BigDecimal("12.20"));
+        productDTO.setMessage("Name and model already exists");
```

Comment:
You can omit this and just validate the product contains the message.
Use constants

Reply:
Fixed